eng/common/pipelines/templates/jobs/docindex.yml

```diff
@@ -1,7 +1,7 @@
 jobs:
   - job: CreateDocIndex
     pool:
-      vmImage: windows-2019
+      vmImage: windows-2022
```

Comment:
@sima-zhu we need to think through how we want to handle eng/common, as we will want to exclude it in all PRs other than the tools repo and in the tools repo we will need to take those extra steps to sync.

Reply:
I wonder if we should include some sort of override message in the commit/PR that allows our tooling to skip enforcement? Could be error prone though.

Reply:
For PRs other than tools repo, we are able to exclude the eng/common in our automation.

Reply:
> I wonder if we should include some sort of override message in the commit/PR that allows our tooling to skip enforcement? Could be error prone though.

The best I can think of is to add something like in the scanned file, we defined the section like:
```
## Skip migration overrides
vmImage: windows-2022
## End of the skip enforcement
```

Our automation script will skip the section defined like this.